templates/devspace.yaml.tpl

```diff
@@ -77,7 +77,9 @@ vars:
   BOX_REPOSITORY_URL: $(yq -r '.storageURL' "$HOME/.outreach/.config/box/box.yaml")
 
 
-  DLV_PORT: 42097
+  DLV_PORT:
+    source: env
+    default: 42097
```

Comment:
Does it Just Work to set a port via an environment variable? Like, is it communicated to the running application correctly?

I see it plumbed through in a few places, and it would be great if this more or less worked.

Reply:
Yes. That is entirely my understanding of it.